js/views/userPageVw.js

```diff
@@ -812,6 +839,17 @@ module.exports = baseVw.extend({
     }
   },
 
+  renderReviews: function (model) {
+    model = model || [];
+    this.reviewsVw = new reviewsView({
```

Comment:
In what case is it called multiple times? I don't see renderFollowing() and others being removed too. I thought registerChild() is taking care of it automatically?